docs/general/fields.md

```diff
@@ -29,3 +29,5 @@ render json: @user, fields: { users: [:access_token] }
 ```
 
 Where `users` is the JSONAPI type.
+
+Casing for multi-word fields should match the casing specified for `ActiveModelSerializers.config.key_transform`.
```

Comment:
@NullVoxPopuli Can you clarify with an example and explanation of how to thing about which casing to choose?

e.g.

```ruby
# BAD, will not work because.... key_transform is x, field name is y, adapter is z, serializer attribute is q, etc.
render json: search_result, fields: { 'event-attendance' => [:attendee_name] }

# GOOD, will work because...key_transform is x, field name is y, adapter is z, serializer attribute is q, etc.
render json: search_result, fields: { 'event-attendance' => ['attendee-name'] }
```